scripts/st2_bootstrap.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
```

Comment:
st2bootstrap-deb.sh please. I am coding it in the ubuntu/deb doc. We will have st2bootstrap-centos.sh, and st2bootstrap.sh selector.